base_kanban_stage/tests/test_base_kanban_abstract.py

```diff
@@ -2,30 +2,32 @@
 # Copyright 2019 Eficent Business and IT Consulting Services S.L.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
+from collections import deque
+
 from odoo import api, models
 from odoo.tests.common import TransactionCase
 
 
-class BaseKanbanAbstractTester(models.Model):
+class BaseKanbanAbstractTester(models.AbstractModel):
     _name = "base.kanban.abstract.tester"
+    _description = "base kanban abstract tester"
     _inherit = "base.kanban.abstract"
 
 
 class TestBaseKanbanAbstract(TransactionCase):
     def _init_test_model(self, model_cls):
         model_cls._build_model(self.registry, self.cr)
         model = self.env[model_cls._name]
-        # setup_models():
         model._prepare_setup()
         model._setup_base()
         model._setup_fields()
-        # init_models():
         model._setup_complete()
-        model._auto_init()
-        model.init()
+        self.registry._post_init_queue = deque()
         while self.registry._post_init_queue:
             func = self.registry._post_init_queue.popleft()
             func()
+        model._auto_init()
+        model.init()
```

Comment:
why? that doesn't make sense. first init, then post_init. that's why it's called 'post' and not 'pre'

Reply:
> why? that doesn't make sense. first init, then post_init. that's why it's called 'post' and not 'pre'

hello @MiquelRForgeFlow,
 Need to do registry, first otherwise, it shows a warning, like this >> (cursor closed), and some checks are failed.

Reply:
@AaronHForgeFlow  it would be nice if could you review it here?